source/_patterns/00-atoms/components/nav-linked-item.mustache

```diff
@@ -1,27 +1,15 @@
 <li class="{{classes}}">
 
-  {{#loginControl}}
```

Comment:
I'm not sure that we need the loginControl any more. We can create a followup ticket to clean this up?

Reply:
No action needed. I will create a ticket.